app/src/main/java/com/projectit3048c/dto/FoodSpecimen.kt

```diff
@@ -1,4 +1,9 @@
 package com.projectit3048c.dto
 
-data class FoodSpecimen(var id: Int=0, var name: String="", var description: String="", var calories: Int=0, var specimenID: String="", var date: String=""){
+data class FoodSpecimen(var id: Int=0,
```

Comment:
Long class headers should have parameters in a separate line.
[https://kotlinlang.org/docs/coding-conventions.html#class-headers](url)